chains/evm/contracts/ccvs/components/BaseVerifier.sol

```diff
@@ -17,9 +19,11 @@ abstract contract BaseVerifier is ICrossChainVerifierV1, ITypeAndVersion {
 
   error CursedByRMN(uint64 destChainSelector);
   error InvalidDestChainConfig(uint64 destChainSelector);
+  error DestGasCannotBeZero(uint64 destChainSelector);
   error InvalidAllowListRequest(uint64 destChainSelector);
   error SenderNotAllowed(address sender);
   error CallerIsNotARampOnRouter(address caller);
+  error InvalidDestination(uint64 destChainSelector);
```

Comment:
Would DestinationNotSupported be clearer? InvalidDestination is similar to InvalidDestChainConfig.